torch/_export/non_strict_utils.py

```diff
@@ -85,15 +86,14 @@ def fakify(
         raise ValueError(f"Unsupported input type {type(t)}")
     n_dims = len(t.shape)
     symbolic_context = StatelessSymbolicContext(
-        dynamic_sizes=[DimDynamic.STATIC] * n_dims,
+        dynamic_sizes=[DimDynamic.DYNAMIC] * n_dims,
         constraint_sizes=[None] * n_dims,
     )
     t_id = id(t)
     assert mode.shape_env is not None
     if t_id in t_constraints:
         for i, constraint in t_constraints[t_id].items():
             symbolic_context.constraint_sizes[i] = constraint.constraint_range
```

Comment:
So what about static dims? They will have a singleton constraint range?

Reply:
Yepp, but we will convert them into static constraints at some point, just not here (I think before we run into any guards).

Reply:
Yeah that's right, there's [some code](https://github.com/pytorch/pytorch/blob/416a2edd7c494bc6d739c27ddd538cf2d1a94d74/torch/fx/experimental/symbolic_shapes.py#L2629-L2639) that'll convert these into static later on though, so they won't have symbols.

But ok, looks like this will still create a sympy.Integer instead of an int, so this is probably responsible for all the is_number checks.